os/include/tinyara/sched.h

```diff
@@ -521,6 +522,13 @@ struct tcb_s {
 	FAR struct tcb_s *flink;	/* Doubly linked list                  */
 	FAR struct tcb_s *blink;
 
+#ifdef CONFIG_BINARY_MANAGER
+	/* Fields used to support binary list management ***************************** */
+
+	FAR struct tcb_s *bin_flink;	/* Doubly linked list				   */
+	FAR struct tcb_s *bin_blink;
+#endif
```

Comment:
Without recovery case, are they used?

Reply:
Yes. they are used when binary manager searches and kill task/threads for binary update.